src/odin/commands/status.rs

```diff
@@ -0,0 +1,49 @@
+use crate::server::ServerInfo;
+use crate::utils::environment::fetch_var;
+use crate::utils::parse_arg_variable;
+use clap::ArgMatches;
+use log::info;
+use std::env;
+
+pub fn fetch_public_address() -> String {
+  let current_ip = env::var("ADDRESS").unwrap_or_else(|_| {
+    reqwest::blocking::get("https://api.ipify.org")
+      .unwrap()
+      .text()
+      .unwrap()
+  });
+  let current_port: u16 = fetch_var("PORT", "2456").parse().unwrap();
+  format!("{:?}:{:?}", current_ip, current_port + 1)
+}
+
+fn parse_address(args: &ArgMatches) -> String {
```

Comment:
I think it would make more sense if `parse_address` and `fetch_public_address` returned an `std::new::SocketAddrV4`

Reply:
huh, good call, then we can error if its an invalid ipv4 address.